components/Rating/rating.css

```diff
@@ -0,0 +1,23 @@
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 1rem;
+}
+
+.rating input {
+  display: none;
+}
+
+label {
```

Comment:
Do not select the direct label tag as it will affect other labels as well

Reply:
Agree on that, you might want to create more labels in the future. Selecting like this might cause problems.